### PR TITLE
Sidebar

### PR DESCRIPTION
**Pull request by knownasilya**
Pulled out some of the initial changes from #490 to get it in.

twiddle-sidebar
Screen Shot 2020-02-29 at 5 11 10 PM
Screen Shot 2020-02-29 at 5 11 19 PM

### CODE REVIEW

File: app/components/file-tree.js [hunk 1]
```diff
@@ -99,22 +99,18 @@ export default Component.extend({
       this.jsTreeActionReceiver.send('toggleNode', node.id);
     },
 
-    didBecomeReady() {
+    didBecomeReadyOptional() {
```
Comment (Gaurav0) [inline app/components/file-tree.js]: Ugh. Can't believe I wrote this. :)

File: app/styles/_toolbar.scss [hunk 2]
```diff
@@ -14,70 +9,36 @@
     height: $topbar-height;
   }
 
-  @media (max-width: $screen-md-min) {
```
Comment (Gaurav0) [inline app/styles/_toolbar.scss]: Is this still responsive?

File: app/components/file-tree.js (hunk 1 above)
Comment (knownasilya) [reply app/components/file-tree.js]: We all have these moments :D

File: app/styles/_toolbar.scss (hunk 2 above)
Comment (knownasilya) [reply app/styles/_toolbar.scss]: No, that is my next task haha.

**Pull request merged**